<?xml version="1.0"?>
<launch>
    <!-- launch gazebo -->
    <include file="$(find glenn_description)/launch/glenn.launch">
        <arg name="use_rviz" value="false"/>
        <arg name="use_steering" value="false"/>
    </include>

    <rosparam command="load" file = "$(find config)/params/glenn_params.yaml"/>
    <rosparam command="load" file = "$(find autonomy)/params/autonomy.yaml" ns="autonomy"/>

    <node name="transit_node" pkg="autonomy" type="transit_node.py" respawn="true" launch-prefix="xterm -e"/>
    <node name="cmd_to_wheel_speeds_node" pkg="autonomy" type="cmd_to_wheel_speeds_node.py"/>
    <node name="autonomy_node" pkg="autonomy" type="autonomy_node.py" launch-prefix="xterm -e"/>
</launch>
